sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2052,7 +2052,7 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
 
           val fileStatus2 = fs.getFileStatus(tablePath)
           if (ignore) {
-            assert(fileStatus2.getPermission().toString() == "rwxr-xr-x")
+            assert(fileStatus2.getPermission().toString() != "rwxrwxrwx")
```

Comment:
This is a loose condition. I think it is rarely to have default file permission as `rwxrwxrwx`. Another option is not to check permission when `ignore` is true.